client/src/testSetup.js

```diff
@@ -0,0 +1,23 @@
+const { JSDOM } = require("jsdom");
+const jsdom = new JSDOM("<!doctype html><html><body></body></html>");
+const { window } = jsdom;
+
```

Comment:
You can add the enzyme configuration snippet here and remove unnecessary statements from `AnswerList.spec.tsx` file.

```js
const { configure } = require("enzyme");
const Adapter = require("enzyme-adapter-react-16");
configure({ adapter: new Adapter() });
```